packages/charts/src/state/selectors/get_screen_reader_summary.ts

```diff
@@ -39,11 +39,11 @@ export const getScreenReaderSummarySelector = createCustomCachedSelector(
     }
 
     // Get chart-specific screen reader data
```

Comment:
nit: do you think we should update the comment?

```suggestion
    // Get chart-specific screen reader items
```